includes/general.php

```diff
@@ -1132,13 +1138,8 @@ function pods_shortcode_run( $tags, $content = null ) {
 	// phpcs:ignore
 	echo $content;
 
-	if ( ! $is_singular && 0 < $found && true === $tags['pagination'] && in_array(
-		$tags['pagination_location'], array(
-			'after',
-			'both',
-		), true
-	) ) {
-		echo $pod->pagination( $tags['pagination_label'] );
+	if ( $tags['pagination'] && in_array( $tags['pagination_location'], array( 'after', 'both', ), true ) ) {
```

Comment:
```suggestion
	if ( $pagination && in_array( $tags['pagination_location'], array( 'after', 'both', ), true ) ) {
```

Reply:
This is invalid, `$pagination` could also be an empty array.